### ISSUE TITLE
adapt-contrib-media Media and Audio don't have descriptive aria-label

### ISSUE THREAD

**Issue by simondate** [adaptlearning/adapt_framework#2822]
### Subject of the issue/enhancement/features

Media elements (video and audio don't have a proper descriptive name, instead just being called Audio player or Video player.

### Your environment
* version (AT/Framework)

5.3.0

### Expected behaviour

Ideally, there should be a descriptive title assigned by the content author

### Actual behaviour

The default title of Audio Player or Video player is set.

It is assigned [here](https://github.com/adaptlearning/adapt-contrib-media/blob/master/libraries/mediaelement-and-player.js#L2410) by this library.

**Comment by moloko** [comment]
at the very least we need be able to localise the existing 'video/audio player' label

@simondate are you saying you think the description should be specific to that particular instance of video? that seems kind of odd to me, I would have thought that would already be covered by body text - otherwise your description of the video would only be available to users of assistive technology, which seems a sort of reverse-discrimination!

**Comment by simondate** [comment]
@moloko This is just me forwarding the issues raised by the accessibility testing. I raised the same point to them but they seemed to think that having something on the actual media source itself would be useful.

It's worth noting that the piece of content sent for testing didn't have a Transcript which I've always used as the accessible alternative to vidoes.
